src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,50 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { useContext } from 'react';
+import { TodoContext } from '../../contexts/TodoContext';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+export const TodoFilter = () => {
+  const { setFilterBy } = useContext(TodoContext);
+  const { searchQuery, setSearchQuery } = useContext(TodoContext);
```

Comment:
Why not join these lines?

`const { setFilterBy, searchQuery, setSearchQuery } = useContext(TodoContext);`

Reply:
Feito!